scripts/generatePackageSwift.swift

```diff
@@ -37,13 +37,6 @@ struct PackageDeps: Codable {
     var clientRuntimePath: String?
 }
 
-// When AWS_SDK_RELEASE_IN_PROGRESS is set, the manifest will require
-// exact versions of smithy-swift and aws-crt-swift no matter what
-// branches or paths are set.
-let releaseInProgress: Bool = {
-    env["AWS_SDK_RELEASE_IN_PROGRESS"] != nil
```

Comment:
If this env var is no longer being read from, then we should remove the place where it is set in our release workflow

Reply:
good call, removed.